config/initializers/octokit.rb

```diff
@@ -0,0 +1,32 @@
+if Rails.env.production?
+  Octokit.configure do |config|
+    config.login = ENV.fetch('GH_USERNAME')
+    config.password = ENV.fetch('GH_PASSWORD')
+  end
+end
+
+class GithubFakeAdapter
+  def initialize(_post); end
+
+  def create_pr
+    Rails.logger.debug('Create a PR')
+  end
+
+  def update_pr
+    Rails.logger.debug('Update the PR')
+  end
+
+  def close_pr
+    Rails.logger.debug('Close the PR')
+  end
+end
+
+module Github
+  def self.adapter
+    if Rails.env.production?
+      GithubAdapter
+    else
+      GithubFakeAdapter
+    end
+  end
+end
```

Comment:
my poins is that these classes will be harder to find in the future, since we usually expect to have each class/module in a file matching with a matching name